### ISSUE TITLE
The doubt of "loss function"

### ISSUE THREAD

**Issue by ugsanjing** [wj926/DomainSeparationNetworks#3]
I think the loss functions is , loss = loss_classification + loss_diff + loss_similarity + loss_recon, however you are achieve loss1 = loss_classification + loss_diff + loss_recon. loss2=loss_similarity,I don't know if I got it wrong, Please point out my fault.THX
